ElectronClient/gui/NoteText.jsx

```diff
@@ -1041,6 +1035,42 @@ class NoteTextComponent extends React.Component {
 
 				indentOrig.call(this);
 			};
+
+			// Delete a list markup (e.g. `- `) from an empty list item on hitting Enter.
+			// (https://github.com/laurent22/joplin/pull/2772)
+			this.editor_.editor.commands.addCommand({
+				name: 'enter',
+				bindKey: 'Enter',
```

Comment:
Are you sure about this?  My original comment was more of a question, but this is better if it works.

Reply:
Yes, they are equivalent.

https://github.com/ajaxorg/ace/blob/c6aa009c9fd12822222b36a81387d91c60c6e8d2/lib/ace/keyboard/hash_handler.js#L91-L95